servo-core/src/test/java/com/netflix/servo/util/ThreadCpuStatsTest.java

```diff
@@ -0,0 +1,59 @@
+/**
+* Copyright 2014 Netflix, Inc.
```

Comment:
change the date to 2019

Reply:
Agreed although I think it should now be a range ie. `2014-2019`

Reply:
Well some of the earlier files are copyright 2013. However i think 2019 is correct here, because the file didn't exist before 2019.

Reply:
Yes, agreed. I was thinking that was the existing copyright year for this file but, as you said, it didn't exist before.